rundeck_exporter.py

```diff
@@ -163,6 +164,19 @@ class RundeckMetricsCollector(object):
                             nargs='+',
                             required=False
                             )
+    args_parser.add_argument('--rundeck.projects.executions.job-option-labels',
```

Comment:
What about using snake_case and also changing it to something like `--rundeck.projects.executions.filter_job_options`?